### ISSUE TITLE
Error launching gulp serve-dev

### ISSUE THREAD

**Issue by MakinWavz** [johnpapa/generator-hottowel#44]
Hi John,

I just started your PluralSight Course - Play by Play: Learning AngularJS With Ken Cenerelli and John Papa.

I have tried generator-hottowel on both windows 7 and Ubuntu 14.04.1 LTS and receive the same error when I launch  **gulp serve-dev**

node - v0.12.0
npm - 2.5.1
yo - 1.4.5
bower - 1.3.12
gulp - 3.8.11

-Chris

```javascript
 Creating an AngularJS $templateCache
/home/chris/hello/node_modules/gulp-less/index.js:68
    }).done(undefined, cb);
       ^
TypeError: undefined is not a function
'''

**Comment by johnpapa** [comment]
i just tried a fresh install and it worked for me. try clearing your npm cache. might have a corrupted gulp-less file
`npm cache clear`

**Comment by MakinWavz** [comment]
No luck - i tried clearing the npm cache on both platforms, but no change - I am going to start fresh and will see if I have a better result

**Comment by danielz1985** [comment]
I observed the same problem with the configuration of MakinWavz.
"gulp-less" was version 2.0.1, changing it to 3.0.1 in package.json solved it for me.
Refer to: https://github.com/plus3network/gulp-less/issues/140

**Comment by johnpapa** [comment]
this is now fixed in 0.2.7
